scripts/maintenance/migrate_project/src/cli.py

```diff
@@ -0,0 +1,79 @@
+from pathlib import Path
+
+import typer
+from db import (
+    db_connection,
+    get_project_and_files_to_migrate,
+    insert_file_meta_data,
+    insert_projects,
+)
+from models import Settings
+from r_clone import assemble_config_file, sync_file
+
+
+def main(config: Path):
+    assert config.exists()
+    settings = Settings.load_from_file(config)
+    typer.echo(f"Detected settings:\n{settings.json(indent=2)}\n")
+
+    assert settings.source.s3.endpoint is not None
+    r_clone_config_path = assemble_config_file(
+        # destination is AWS S3
+        aws_access_key=settings.destination.s3.access_key,
```

Comment:
Possible enhancement: Allow all providers currently used in oSparc (aws, ceph, minio) in all permutations.

Reply:
Will not have it now. If required I will code it.